tests/test_packages/test_connections/test_ledger/test_ledger_api.py

```diff
@@ -174,6 +174,7 @@ async def test_get_balance(
 
 @pytest.mark.integration
 @pytest.mark.ledger
+@pytest.mark.flaky(reruns=2, reruns_delay=5)
```

Comment:
![image](https://user-images.githubusercontent.com/35799987/160844854-d52527a5-1b4b-441e-8070-43a2b48770be.png)

This test regularly fails due to a network dependency on fetchai.

Reply:
Yes we will replace this soon. So for now let's leave as is.